test_conformance/images/clCopyImage/test_loops.cpp

```diff
@@ -88,6 +100,21 @@ int test_image_type( cl_device_id device, cl_context context, cl_command_queue q
             imageType = CL_MEM_OBJECT_IMAGE3D;
             break;
     }
+    else if (testMethod == k1DBuffer)
```

Comment:
The switch case above doesn't have a default so this is failing compilation with "enumeration values 'k1DBuffer', 'k1DTo1DBuffer', and 'k1DBufferTo1D' not handled in switch "
And here again, not seeing an if to match the else if cascade so might be better to move everything inside the switch?

Reply:
Sorry about that, it was a rebase error on my side.